org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/ParameterizedExpressionsTestLanguageGrammarAccess.java

```diff
@@ -61,22 +61,25 @@
 		
 		//FunctionDeclaration <Yield>:
 		//	=> ({FunctionDeclaration}
-		//	-> FunctionImpl <Yield,Yield,Expression=false>) => ';'?;
+		//	-> FunctionImpl<Yield,Yield,Expression=false>) => ';'?;
 		@Override public ParserRule getRule() { return rule; }
 		
-		//=> ({FunctionDeclaration} -> FunctionImpl <Yield,Yield,Expression=false>) => ';'?
+		//=> ({FunctionDeclaration}
```

Comment:
@cdietrich What do you think about the space after the predicates `=> (..)` and `-> (..)` - I've slight preference towards noSpace for that location.

Reply:
me too